recipes/r-rbiom/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set version = '1.0.2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rbiom
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/rbiom_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/rbiom/rbiom_{{ version }}.tar.gz
+  sha256: bea0ad0262fe5fe85967371164317ecd440b55a4943fa1c9aa35c1f1fb8ae17d
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - r-base
+    - r-r.utils
+    - r-rcpp
+    - r-rcppparallel
+    - r-magrittr
+    - r-openxlsx
+    - r-plyr
+    - r-rjson
+    - r-slam
+
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
+    - r-r.utils
+    - r-rcpp
+    - r-rcppparallel
+    - r-magrittr
+    - r-openxlsx
+    - r-plyr
+    - r-rjson
+    - r-slam
+
+test:
+  commands:
+    - $R -e "library('rbiom')"           # [not win]
+    - "\"%R%\" -e \"library('rbiom')\""  # [win]
+
+about:
+  home: https://cmmr.github.io/rbiom/index.html
+
+  license: AGPL-3.0-or-later
+  license_family: AGPL
+  license_file: LICENSE

```

Comment:
```suggestion
  license_file:
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/AGPL-3
```

Reply:
This makes it use the `r-base` installed license file; please remove the LICENSE file in this recipe.